examples/logical_enclosure.pp

```diff
@@ -13,32 +13,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ################################################################################
+
```

Comment:
Let's update the license date in all the modified files

Reply:
Done